coin/models_test.go

```diff
@@ -37,3 +37,48 @@ func TestGetCoinForId(t *testing.T) {
 		})
 	}
 }
+
+func TestGetCoinExploreURL(t *testing.T) {
+	type args struct {
+		addr  string
+		chain string
+	}
+
+	tests := []struct {
+		name    string
+		args    args
+		want    string
+		wantErr bool
+	}{
+		{
+			name: "Test ethereum",
+			args: args{
+				addr:  "token",
+				chain: "ethereum",
+			},
+			want:    "https://etherscan.io/token/token",
+			wantErr: false,
+		},
+		{
```

Comment:
can you add couple of more test cases here? for tron, since it's using two explorers